content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -114,7 +114,7 @@ The name of a Role or a ClusterRole object must be a valid
 
 ### RoleBinding and ClusterRoleBinding
 
-A role binding grants the permissions defined in a role to a user or set of users.
+A RoleBinding grants the permissions defined in a role to a user or set of users.
```

Comment:
I think the existing test is more correct. This intro paragraph address both `RoleBinding` and `ClusterRoleBinding` API objects, and explains the difference between them.